osvr/RenderKit/RenderManagerD3DBase.cpp

```diff
@@ -1338,13 +1338,34 @@ namespace renderkit {
 
         //====================================================================
         // Create the shader resource view.
+        D3D11_TEXTURE2D_DESC colorBufferDesc = { 0 };
+        params.m_buffer.D3D11->colorBuffer->GetDesc(&colorBufferDesc);
+        DXGI_FORMAT shaderResourceViewFormat = DXGI_FORMAT_R8G8B8A8_UNORM;
+        switch (colorBufferDesc.Format) {
+        case DXGI_FORMAT_B8G8R8A8_TYPELESS:
+            shaderResourceViewFormat = DXGI_FORMAT_B8G8R8A8_UNORM;
+            break;
+        case DXGI_FORMAT_R8G8B8A8_TYPELESS:
+            shaderResourceViewFormat = DXGI_FORMAT_R8G8B8A8_UNORM;
+            break;
+        case DXGI_FORMAT_R8G8B8A8_UNORM:
+            shaderResourceViewFormat = DXGI_FORMAT_R8G8B8A8_UNORM;
+            break;
+        case DXGI_FORMAT_B8G8R8A8_UNORM:
+            shaderResourceViewFormat = DXGI_FORMAT_B8G8R8A8_UNORM;
+            break;
+        default:
+            std::cerr << "osvr::renderkit::RenderManagerD3D11Base::PresentEye - unknown render target texture format. Defaulting to DXGI_FORMAT_R8G8B8A8_UNORM." << std::endl;
```

Comment:
Because this is in presenteye, it would generate a LOT of spam.  The right thing is probably to move the resource creation into the registration code rather than where it is here (somewhere on my todo list already), but for now go ahead and comment out the error message and do the merge.